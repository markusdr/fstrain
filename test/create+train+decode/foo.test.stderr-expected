12 / 25 = 0.4800 correct
