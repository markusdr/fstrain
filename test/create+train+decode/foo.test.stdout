S n u d e l t E
S g e k l e c h s t E
S b e s t r e i k t E
S e r m a e c h t i t E
S e r s c h i e s s t E
S f e r t E
S e r s t e i t E
S b i n d e t E
S v e r s c h u l d e t E
S g e s t i e b t E
S r a l l t E
S g e l a d e t E
S e e h m i t E
S i t E
S v e r r e l l t E
S w u r s t e l t E
S b u l l e r t E
S w i d e r s t r e b t E
S e n t r a t e t E
S u r c h l u e h t E
S g e s p a e n t E
S w e r t e t E
S a u l t i e r t E
S g e s i n t E
S a o s i e r t E
